doc/Operation.md

```diff
@@ -0,0 +1,328 @@
+# Operation\<T>
+
+The Azure SDK for C++ defines an Operation as an abstract class to support a long running operation ([LRO](http://restalk-patterns.org/long-running-operation-polling.html)).
+
+The SDK clients would define a concrete class from the `Operation<T>` which an end users can consume for getting the status of the running operation.
```

Comment:
Would the end-user ever defines these? This doc is for the end-user, I recommend talking directly to them as the audience.

```suggestion
Client libraries define specific concrete `Operation<T>` classes which you, as the end user calling the LRO API, would get back. This allows you to poll and get the status of your request.
```

Reply:
I don't think we have guidelines about the style for writing tech docs, do we?
I prefer not to use the `you ...` and at any case use something like `we ...` :)